### PR TITLE
What works for @armedbear currently

### PR DESCRIPTION
**Pull request by easye**
Initial foray into compatibilty: add a `fix-abcl.lisp`.

### CODE REVIEW

**Comment by easye** [comment]
Until we fall down by missing a STATIC-VECTORS implementation.

File: Lisp-Dep/fix-abcl.lisp [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+                compute-applicable-methods method-qualifiers class-name))
+
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  (loop :for symbol :being :the :symbols :of :clim-mop
+     do (export symbol :clim-mop))
+    (loop :for symbol :being :the :external-symbols :of #:mop
+     do (export symbol #:mop)))
+  (loop :for symbol :being :the :external-symbols :of #:mop
+     do (export symbol #:mop)))
+
```
Comment (dkochmanski) [inline Lisp-Dep/fix-abcl.lisp]: we already use closer mop for that, so this fix is not necessary

File: clim-lisp.asd [hunk 2]
```diff
@@ -4,12 +4,19 @@
...
+                        (#+(or cmu scl excl sbcl openmcl lispworks clisp ecl)
+                           (:file   #+armedbear "fix-abcl"
+                                    #+cmu       "fix-cmu"
+                                    #+scl       "fix-scl"
+                                    #+excl      "fix-acl"
+                                    #+sbcl      "fix-sbcl"
+                                    #+openmcl   "fix-openmcl"
+                                    #+lispworks "fix-lispworks"
+                                    #+clisp     "fix-clisp"
+                                    #+ecl       "fix-ecl")))
```
Comment (dkochmanski) [inline clim-lisp.asd]: this seems to be based on old mcclim version, we do not have most of these files anymore

**Comment by dkochmanski** [comment]
could you try loading newest mcclim? I think that these fixes are not necessary with current codebase

**Comment by dkochmanski** [comment]
also static vectors shouldn't be a dependency anymore with up to date systems (most notably opticl), dependency came from pngload and now it is:
```
(asdf:defsystem :opticl
  :name "opticl"
  :description "A library for representing and processing images"
  :author "Cyrus Harmon <ch-lisp@bobobeach.com>"
  :licence "BSD"
  :serial t
  :default-component-class cl-source-file
  :depends-on (alexandria
               retrospectiff
               zpng
               #-(or clisp abcl clasp) pngload
               #+(or clisp abcl clasp) png-read
               cl-jpeg
               skippy
               opticl-core
               cl-tga)
  :components
  ((:static-file "README.md")
   (:static-file "COPYRIGHT")
   (:file "package")
   (:file "coerce")
   (:file "colors")
   (:file "imageops")
   (:file "invert")
   (:file "transform")
   (:file "convolve")
   (:file "morphology")
   (:file "gamma")
   (:file "shapes")
   (:file "tiff")
   (:file "jpeg")
   (:file "png")
   (:file "pngload")
   (:file "pnm")
   (:file "gif")
   (:file "tga")
   (:file "io")
   (:file "cluster")
   (:file "thresholding"))
  :in-order-to ((test-op (test-op :opticl/test))))
```

so abcl was excluded for some time now (a few months at least afair)

the biggest issue with abcl support will be xlib support I think.

File: Lisp-Dep/fix-abcl.lisp (hunk 1 above)
Comment (easye) [reply Lisp-Dep/fix-abcl.lisp]: will fix

**Comment by easye** [comment]
Closing pull fix until I can experiment with the feedback from 

Thanks for the review @dkochmanski !

**Pull request closed without merging**